src/main/java/com/blazemeter/ciworkflow/CiPostProcess.java

```diff
@@ -283,22 +285,33 @@ protected JSONObject getFunctionalReport(Master master) {
     }
 
     public File makeReportDir(String reportDir) throws Exception {
-        File f = new File(reportDir);
+        File f;
+        File workspaceDir;
+        String reportDirNoNull = reportDir.replace("null" + File.separator, "");
+        if (this.workspaceDir == null) {
+            workspaceDir = File.createTempFile(reportDirNoNull, reportDirNoNull);
```

Comment:
Maybe we need create `temp directory` ?
See `java.nio.file.Files#createTempDirectory` or `com.google.common.io.Files#createTempDir`